gopher/enrichment.py

```diff
@@ -62,22 +64,33 @@ def test_enrichment(
         Download the GO annotations even if they have been downloaded before?
     progress : bool, optional
         Show a progress bar during enrichment tests?
-
+    annotations: pandas.dataFrame(), optional
+        A custom annotations file.
```

Comment:
**nitpik:** This is technically a file 😛

Reply:
Done